### ISSUE TITLE
[IOS] double authentication FaceID and Password

### ISSUE THREAD

**Issue by saulstys** [j3k0/cordova-plugin-purchase#1094]
### system info

```
OSX Catalina 10.15.4
Capacitor 2.1.0. / Ionic 5.2.6
Device: iPhone iOS 13.6.1
Plugin Version: 10.2.0

```

### Expected behavior
When the FaceID is enabled, we should not see the dialog with the apple id password.

### Observed behavior
When i initiate the purchase, i get a dialog asking me to enter my apple password AND the FaceID layer, asking me to click 2 times the right button to pay.

If i click on the button, the purchase is validated but the dialog remains on the screen.

Is there a way to avoid to see the password dialog ?

Thanks

![IOS_purchase](https://user-images.githubusercontent.com/48296431/91731046-2faea400-ebaf-11ea-8529-39f9ba9ecc28.jpeg)

**Comment by j3k0** [comment]
Probably solved by #1092 - can you try it out?

**Comment by saulstys** [comment]
Hi @j3k0 ,
I tried your fix from my Xcode, i don't see the FaceId dialog anymore, but i wasn't seeing it before publishing the app on the appstore.
I'm not sure if the FaceId is visible only for the apps downloaded from Appstore.
